.github/workflows/ansible-test-plugins.yml

```diff
@@ -111,37 +85,31 @@ jobs:
             ansible: stable-2.12
 
     steps:
-
-      - name: Check out code
-        uses: actions/checkout@v2
-        with:
-          path: ansible_collections/community/mysql
-
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python }}
-
-      - name: Install ansible-base (${{ matrix.ansible }})
-        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
-
-      - name: Set MySQL version (${{ matrix.mysql }})
-        run: "sed -i 's/^mysql_version:.*/mysql_version: \"${{ matrix.mysql }}\"/g' ${{ env.mysql_version_file }}"
```

Comment:
This `sed` is different on the main branch. There, it doesn't wrap the version with double quotes and here it does.
```diff
-        sed -i 's/^mysql_version:.*/mysql_version: "${{ matrix.mysql }}"/g' ${{ env.mysql_version_file }}
+          sed -i "s/^mysql_version:.*/mysql_version: $DB_VERSION/g" ${{ env.mysql_version_file }}
```

Reply:
For registering the "reqs" vars, the playbook needs "db_version.version.major <= 5". I feel like I have overlooked how "mysql_version" is used. I'll have a look at it today if I can. Thanks @webknjaz for your investigation!

Reply:
Pleas forget about my previous comment. db_version is set by setup_mysql. Nothing to do with GHA.

Reply:
@laurent-indermuehle click the gear, view logs, i searched by "FAILED:" ignoring stuff with `ignoring` or something like that, after you find the first error, it'll give you a clue how to parse the rest

Reply:
@Andersson007 The issue with "View raw logs" is that all texts are now black on white and color codes are displayed.
But have you tried "View job summary"? It displays only errors. Thought, I can't see if they are ignored or not.

Reply:
@laurent-indermuehle ah, cool, thanks:) i think they has introduced this since i last time tried to view errors there a long time ago - things usually pop up when testing locally. Good to know, thanks

Reply:
> I need to find a way to see errors in the GHA logs. 48000 lines to parse... My browser dies in the process...

Using the built-in search on the job page helps. The UI downloads the logs in chunks. And yes, downloading a raw logs and using something like Vim that loads files by chunks would help.